test/index.test.js

```diff
@@ -75,3 +76,24 @@ t.test('handler', async t => {
       { message: 'Invalid JSON in event body: {invalid-json' })
   })
 })
+
+function assertIsIndexerCompletedPublish(t, snsPublish) {
+  const event = JSON.parse(snsPublish.Message)
+  assertIsIndexerCompletedEvent(t, event)
+}
+
+const ONE_HOUR_MILLISECONDS = 1000 * 60 * 60
```

Comment:
It should take significantly _less_ than this for the test though right?...and I'm pretty sure the test runner would timeout the test if it took this long! Personally I'd use something like 5s.

Reply:
@alanshaw This test was to catch the case where the handling didn't actually take that long, but the calculation of the `startTime` datetime was wrong. i.e. the actual duration in the test was probably less than one second even, but the test still would have failed because `endTime - startTime` was greater than a year (because `startTime` was close to unix epoch instead of now-1sec).